deploy/osd-fedramp-cluster-monitoring-config/config.yaml

```diff
@@ -0,0 +1,10 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+    - key: hive.openshift.io/version-major-minor
+      operator: NotIn
+      values: ["4.5"]
```

Comment:
What's the purpose of this `matchExpression`? This version doesn't exist in FedRAMP right?

```suggestion
```